### DISCUSSION TITLE
Customizable "Right Click"

### DISCUSSION THREAD

**Discussion by SystemError12** [Floorp-Projects/Floorp#1013] [Feature Request]
I wonder if it would be possible to customize the Right-click Menu. For example add entries or remove, and rename those entries to your liking. Same as placeholders/lines to distinguish categories. For example If I only need "Open in New Tab" and maybe "Search" Function, it would be great to remove all the other entries. That is of course just an example. Possibilities are endless and the preferences can vary substantially, depending on who uses the browser. 

Are there any plans to make this Shell-Menu customizable? As in my other Suggestion/FeatureRequest, best example of a browser that does have that option is Vivaldi. Is this even possible with Gecko-Engine? 

Any thoughts?

**Comment by cameronj86** [comment]
Definitely something I'm interested in as I suddenly have to scroll to reach the option that I use most frequently.
